content/openssh/_index.md

```diff
@@ -14,35 +14,75 @@ Portable OpenSSH is a port of OpenBSD's OpenSSH to most Unix-like operating syst
 During key exchange (KEX) negotiation, the Qrypt SDK will generate an additional quantum-secure secret to be prepended to the session key hash inputs. Any conventional KEX algorithm can be enhanced by Qrypt security; a Qrypt-secured algorithm can be identified by the @qrypt.com suffix.
 
-The following sections will cover the two ways of obtaining Qrypt OpenSSH; by either pulling a Docker image and creating a demo container, or building Portable OpenSSH from source and adding a version of the Qrypt SDK with a C wrapper.
+The following sections will cover the two ways of obtaining Qrypt OpenSSH; by either downloading a Docker Compose file and using it to build a demo cluster, or building Portable OpenSSH from source and adding a version of the Qrypt SDK with a C wrapper.
 
-## Instructions to create a demo container
+## Instructions to create a demo cluster
 
 First, visit the [Qrypt portal](https://portal.qrypt.com), make a free account, and generate a keygen token.
+Then, download our Docker Compose file [here](/docker-compose.yaml), and paste your token at the location labeled ***\<PASTE-TOKEN-HERE>***
 
-### Set up sshd terminal:
-- Pull the Docker image: `docker pull qryptdev001/openssh:0.1.0`
-- Run docker container in interactive mode: `docker run -it --entrypoint bash --rm --name democontainer qryptdev001/openssh:0.1.0`
-- Paste your token into the environment: `export TOKEN=<token>`
-- Run sshd: `$(which sshd) -o QryptToken=$TOKEN`
-  - Add the `-d` flag to enable verbose logging and automatically close sshd at the end of the session.
+### Start the Demo Cluster
+In your terminal, run `docker-compose up --build` to build both the sshd server and the ssh/sftp client. The terminal will run the sshd-server and print its debug outputs. To terminate the cluster, press Ctrl+C.
+In a new terminal, run `docker exec -it ssh-client bash` to open an interactive terminal in the ssh-client machine, which is equipped with both ssh and sftp.
 
-### Set up ssh/sftp terminal:
-- Open a second interactive session in the running container: `docker exec -it democontainer bash`
-- Paste your token into the environment: `export TOKEN=<token>`
-- Run ssh or sftp: `<ssh/sftp> -o QryptToken=$TOKEN sftpuser@127.0.0.1`
-  - Add the `-v` flag to enable verbose logging.
-- Log into sftpuser with the password "pass"
+### SSH Demo
+
+Assuming you are now in the ***ssh-client*** container:
+
+```bash
+export TOKEN=<PASTE-TOKEN-HERE> # Paste your token into the environment
+ssh -v -o QryptToken=$TOKEN sshuser@sshd-server.com # Log in to sshd-server.com as the user "sshuser"
+```
+
+The default password for sshuser is "pass".
+
+To show that we're now logged into the server:
+```bash
+bash # Change to a shell that shows the current user, the machine name, and the current working directory
+exit # Exit Bash (when you're done on the server)
+exit # Exit sshd-server
+```
+
+### SFTP Demo
+
+Assuming you are still in the ***ssh-client*** container:
```

Comment:
"The following commands are to be run in the ***ssh-client*** container. The verbose printout should printout a line  indicating the kex algorithm used is curve25519-sha256@qrypt.com which is Qrypt's new algorithm".